example/simple-request-anonymized-manual.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Scenario:
+ * Generate requests towards specific domain
+ **/
+var trafficSimulator = require('flexbench');
+
+
+function runTest() {
+    trafficSimulator.debugMode(true);
+    trafficSimulator.testDuration(5);//-1 for infinite run
+    trafficSimulator.workers(1);
+    trafficSimulator.clients(2)
+    trafficSimulator.throttleRequests_bps(50000);//-1 for no throttling
+    trafficSimulator.randomDelayBetweenRequests('0.5-1.1');
+
+    //Anonymized 
+    trafficSimulator.setAnonymization(true);
+    trafficSimulator.setAnonymizationManualConfig('../example/anonymizer-config.json');
```

Comment:
Since the files are under the same folder (and when trying, you will be already under that path), the path `../example` might not be needed